platform/android/java/lib/src/org/godotengine/godot/input/GodotInputHandler.java

```diff
@@ -368,4 +394,90 @@ private int findJoystickDevice(int device_id) {
+	 */
+	private boolean handleMouseEvent(MotionEvent event, final boolean isCaptured) {
+		if (event.getActionMasked() == MotionEvent.ACTION_HOVER_ENTER ||
+				event.getActionMasked() == MotionEvent.ACTION_HOVER_MOVE ||
+				event.getActionMasked() == MotionEvent.ACTION_HOVER_EXIT) {
+			final int x = Math.round(event.getX());
+			final int y = Math.round(event.getY());
+			final int type = event.getAction();
+			queueEvent(new Runnable() {
+				@Override
+				public void run() {
+					GodotLib.hover(type, x, y);
+				}
+			});
+			return true;
+		} else if (event.getAction() == MotionEvent.ACTION_BUTTON_PRESS || event.getAction() == MotionEvent.ACTION_BUTTON_RELEASE) {
+			final float x = event.getX();
+			final float y = event.getY();
+			final int buttonMask = event.getButtonState();
+			final int action = event.getAction();
+			queueEvent(new Runnable() {
+				@Override
+				public void run() {
+					GodotLib.mouseEvent(action, buttonMask, x, y, isCaptured);
+				}
+			});
+			return true;
+		} else if (event.getAction() == MotionEvent.ACTION_SCROLL) {
+			final float x = event.getX();
+			final float y = event.getY();
+
+			if (event.getAxisValue(MotionEvent.AXIS_VSCROLL) > 0.0f)
+				queueEvent(new Runnable() {
+					@Override
+					public void run() {
+						GodotLib.mouseEvent(MotionEvent.ACTION_BUTTON_PRESS, MOUSE_WHEEL_UP, x, y, isCaptured);
+						GodotLib.mouseEvent(MotionEvent.ACTION_BUTTON_RELEASE, MOUSE_WHEEL_UP, x, y, isCaptured);
+					}
+				});
+			else {
+				queueEvent(new Runnable() {
+					@Override
+					public void run() {
+						GodotLib.mouseEvent(MotionEvent.ACTION_BUTTON_PRESS, MOUSE_WHEEL_DOWN, x, y, isCaptured);
+						GodotLib.mouseEvent(MotionEvent.ACTION_BUTTON_RELEASE, MOUSE_WHEEL_DOWN, x, y, isCaptured);
+					}
+				});
+			}
+			if (event.getAxisValue(MotionEvent.AXIS_HSCROLL) > 0.0f) {
+				queueEvent(new Runnable() {
+					@Override
+					public void run() {
+						GodotLib.mouseEvent(MotionEvent.ACTION_BUTTON_PRESS, MOUSE_WHEEL_RIGHT, x, y, isCaptured);
+						GodotLib.mouseEvent(MotionEvent.ACTION_BUTTON_RELEASE, MOUSE_WHEEL_RIGHT, x, y, isCaptured);
+					}
+				});
+			} else {
+				queueEvent(new Runnable() {
+					@Override
+					public void run() {
+						GodotLib.mouseEvent(MotionEvent.ACTION_BUTTON_PRESS, MOUSE_WHEEL_LEFT, x, y, isCaptured);
+						GodotLib.mouseEvent(MotionEvent.ACTION_BUTTON_RELEASE, MOUSE_WHEEL_LEFT, x, y, isCaptured);
+					}
+				});
+			}
+			return true;
+		} else if (event.getAction() == MotionEvent.ACTION_MOVE) {
+			final float x = event.getX();
+			final float y = event.getY();
+			final int buttonMask = event.getButtonState();
+			final int action = event.getAction();
+			queueEvent(new Runnable() {
+				@Override
+				public void run() {
+					GodotLib.mouseEvent(action, buttonMask, x, y, isCaptured);
+				}
+			});
+			return true;
```

Comment:
This is no longer needed as it'll be handled by the logic in the moved `onTouchEvent`.